change/@microsoft-teams-js-ee0877f5-a59f-4ca2-a51c-a21f9c37d769.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Adding isSupported work for chat and conversations",
```

Comment:
Note for everyone else: This isn't the level of detail we would want in the final version of the changelog but we plan to aggregate the isSupported work on the capabilities that we get this done in at the very end.

Reply:
Thanks that was my question when reviewing this PR